buildpipeline/Core-Setup-RHEL7-x64.json

```diff
@@ -164,6 +183,9 @@
     },
     "GITHUB_PASSWORD": {
       "value": "PassedViaPipeBuild"
+    },
+    "PB_DockerImageName": {
```

Comment:
This convention was introduced in CoreFX during dev/eng to avoid colliding with properties in MSBuild through the env vars VSTS sets. In CoreFX, there are many of them that aren't passed in from pipeline.json.

I agree it would make our lives easier if we also had the convention you're describing.